### DISCUSSION TITLE
failer.gd indent wrong?

### DISCUSSION THREAD

**Discussion by Foxomat** [bitbrain/beehave#25] [Q&A] [answered]
In the failer.gd script, the last `return FAILURE` line is inside of the for loop, is that correct? If it has no children, it would return nothing.

failer.gd
```
extends Decorator

class_name AlwaysFailDecorator, "../../icons/fail.svg"


func tick(action, blackboard):
	for c in get_children():
		var response = c.tick(action, blackboard)
		if response == RUNNING:
			return RUNNING
		return FAILURE
```

**Comment by bitbrain** [comment, marked as answer]
@Foxomat great catch! Can you [raise an issue here]( https://github.com/bitbrain/beehave/issues) for this? Also, feel free to raise a PR to fix this.

**Comment by Foxomat** [comment]
Oh I see, another change snuck through in my commit (that I thought I discarded), but it's actually another fix. You can have a look at the [commit](https://github.com/bitbrain/beehave/commit/166bd6b859ffb222a720f2818612be3e6354ee1f), I changed another indent in one of the examples in the `readme.md` (was a statement after return).
